pkg/autoops/api/api_test.go

```diff
@@ -921,13 +921,23 @@ func createAutoOpsService(c *gomock.Controller, db storage.Client) *AutoOpsServi
 	mysqlClientMock := mysqlmock.NewMockClient(c)
 	featureClientMock := featureclientmock.NewMockClient(c)
 	accountClientMock := accountclientmock.NewMockClient(c)
-	ar := &accountproto.GetAccountResponse{
-		Account: &accountproto.Account{
-			Email: "email",
-			Role:  accountproto.Account_VIEWER,
+	ar := &accountproto.GetAccountV2ByEnvironmentIDResponse{
```

Comment:
Just confirming, is there another type of getting an account response?
If not, we could just use `GetAccountV2Response`.

Reply:
@cre8ivejp 

Product code uses the `GetAccountV2ByEnvironment` API because we don't receive the organizationID from the request. (We receive the environmentID.)

https://github.com/bucketeer-io/bucketeer/pull/651/files#diff-13112ed94680ecaec33373531ff5c65e48cecae142657203e04d967a185684d5R1459